### ISSUE TITLE
GNUstep porting

### ISSUE THREAD

**Issue by mogui** [swisspol/GCDWebServer#12]
It would be nice to have this project running on other platforms beside OS X. It could be accomplished by using [GNUstep](http://www.gnustep.org/).
I've tried to make things work, but with poor success.
Main problem is the CFNetwork (mainly calls to CFNetServices ) and CoreServices dependancy.

I think that could be possible to port CFNetServices function calls to NSNetServices ( since it is implemented in GNUstep) and see what we could do for CoreServices call.

I'll give it a try when I can.
I opened an issue to let you know about that and ask you if there's something I've to know before starting working on it (maybe some hard reason why dependency on those framework cannot be removed)
And of course recruiting someone who would help me do it :D 

Bests
